generators/generator-adaptive-bot/generators/app/templates/dotnet/functions/botName.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.10" />
     <PackageReference Include="Microsoft.AspNetCore.OData" Version="7.5.2" />
     <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Runtime" Version="4.12.0-daily.preview.20210126.206273.e467c25" />
+    <PackageReference Include="Microsoft.Bot.Builder.Runtime" Version="4.12.0-daily.preview.20210208.211224.35d8ba0" />
```

Comment:
Probably want to undo this since its already in my PR with RC0

Reply:
Or make it RC0 so git figures things out. also Microsoft.Bot.Builder.Integration.Runtime.